.travis.yml

```diff
@@ -42,6 +42,12 @@ jobs:
       script:
         - make itest backend=neutrino
 
+    - name: Btcd Integration ARM
+      script:
+        - GOARM=7 GOARCH=arm GOOS=linux CGO_ENABLED=0 make btcd build-itest
+        - file lnd-itest
+        - GOARM=7 GOARCH=arm GOOS=linux CGO_ENABLED=0 make itest-only
+      arch: arm64
```

Comment:
Open question: should we only do ARM 32 (since that's more popular atm due to rasp pis), or both? In the original issue, I linked a way to force it (the VM the job is running on) into 64-bit mode.

Reply:
Scratch this, re-read the PR body.